ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/PageSet.kt

```diff
@@ -29,6 +29,7 @@ enum class PageSet(
             Locale.ENGLISH,
             Locale("es"),
             Locale.FRENCH,
+            Locale("hi"),
```

Comment:
I actually think you need to add it to the `TRAINING` PageSet as well. that controls whether the user can open up the training tutorial from the nav drawer